src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/NewUser.vue

```diff
@@ -1,78 +1,108 @@
-            class="max-width-300"
-            :rules="phoneRules"
-            @keypress="isNumber($event)"
-          ></v-text-field>
-          <ECEHeader title="Your ECE registration" />
-          <p class="small mt-2">
-            To add an existing ECE certificate to your account to manage it online, you must enter the number now to link it to your account.
-          </p>
-          <p class="small mt-5">Do you have, or ever had, an ECE certificate in British Columbia?</p>
-          <v-radio-group v-model="eceCertificateStatus" :rules="radioRules">
-            <v-radio label="Yes" :value="true"></v-radio>
-            <v-radio label="No" :value="false"></v-radio>
-          </v-radio-group>
-
-          <v-text-field
-            v-if="eceCertificateStatus === true"
-            v-model="eceRegistrationNumber"
-            label="Your ECE Registration Number"
-            variant="outlined"
-            color="primary"
-            class="max-width-300"
-            :rules="eceRegistrationRules"
-            @keypress="isNumber($event)"
-          />
-          <v-checkbox v-model="hasAgreed" class="mt-2" label="" color="primary" :rules="hasAgreedRules">
-            <template #label>
-              <div>
-                I have read and accept the
-                <router-link to="/new-user/terms-of-use">Terms of Use</router-link>
-              </div>
-            </template>
-          </v-checkbox>
-        </div>
-      </v-form>
-      <v-row>
-        <v-btn rounded="lg" color="primary" class="mr-2" @click="submit">Save and continue</v-btn>
-        <v-btn rounded="lg" variant="outlined" @click="logout">Cancel</v-btn>
-      </v-row>
-    </div>
+      </v-col>
+
+      <!-- Form Section -->
+      <v-col cols="12">
+        <v-form ref="form" validate-on="input">
+          <v-row class="ga-2">
+            <v-col cols="12">
+              <ECEHeader title="Contact Information" />
+              <p class="small mt-2">We'll use this to contact you about your account and updates about your application or certificate.</p>
+            </v-col>
+
+            <!-- Email Field -->
+            <v-col cols="12" md="6">
+              <v-text-field
+                v-model="email"
+                label="Email"
+                variant="outlined"
+                color="primary"
+                type="email"
+                :rules="[Rules.required(), Rules.email('Enter your email in the format \'name@email.com\'')]"
+                class="mt-5"
+              ></v-text-field>
+            </v-col>
+
+            <!-- Phone Field -->
+            <v-col cols="12" sm="6">
+              <v-text-field
+                v-model="phoneNumber"
+                label="Phone number"
+                variant="outlined"
+                color="primary"
+                :rules="[Rules.required(), Rules.phoneNumber()]"
+                @keypress="isNumber($event)"
+              ></v-text-field>
+            </v-col>
```

Comment:
Do the email and phone number cols settings work as expected? 

From how I read it, in a large screen, email and phone number would appear side by side like this 

![image](https://github.com/user-attachments/assets/863d639d-0a0c-4d25-af24-927b9069b4c8)

then between the md size screen and small size screen email would span the entire row and phone number would take half like this 

![image](https://github.com/user-attachments/assets/2847511a-1f94-4cdd-84a9-aa1b3d483716)

Reply:
somehow it was not happening before, but I put those into different v-rows, solved the problem. but I believe the first solution 'max-width' was a better way to handle this as we did not need columns and rows here. it also was supporting responsive.

Reply:
@farzadnadiri sorry noticed I put the wrong photo for the example. Updated it

Reply:
yeah putting it into separate v-rows fixes the problem. 

It gets funky with cols when we mix and match the numbers expecting it to show up in separate rows.